### PR TITLE
74876 - [CDP][FSR] - Design System V3 web component update identification - CDP ONLY

### PR DESCRIPTION
**Pull request by OptionSelect**
## Summary
Add missing attribute to components that did not have it in CDP.

## Related issue(s)

department-of-veterans-affairs/va.gov-team#74876

## What areas of the site does it impact?

CDP

### Quality Assurance & Testing

- [ X ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ X ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ X ] Linting warnings have been addressed
- [ X ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ X ] Screenshot of the developed feature is added
- [ X ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ X ] Browser console contains no warnings or errors.
- [ X ] Events are being sent to the appropriate logging solution
- [ X ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ X ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/combined-debt-portal/combined/components/ComboAlerts.jsx [hunk 1]
```diff
@@ -50,14 +52,14 @@ ComboAlert.Zero = () => {
       <ul>
         <li>
           <strong>For benefit debts</strong>, call the Debt Management Center
-          (DMC) at <va-telephone contact="8008270648" />(
-          <va-telephone tty contact="711" />
+          (DMC) at <va-telephone contact="8008270648" uswds />(
```
Comment (josephrlee) [inline src/applications/combined-debt-portal/combined/components/ComboAlerts.jsx:55]: @OptionSelect , fyi, there's no v3 version of this component. So I don't think adding the `uswds` prop is necessary to `va-telephone`.

File: src/applications/combined-debt-portal/combined/components/LoadingSpinner.jsx [hunk 2]
```diff
@@ -3,7 +3,10 @@ import PropTypes from 'prop-types';
 
 const LoadingSpinner = ({ margin }) => (
   <div className={`vads-u-margin--${margin}`}>
-    <va-loading-indicator message="Please wait while we load the application for you." />
+    <va-loading-indicator
```
Comment (josephrlee) [inline src/applications/combined-debt-portal/combined/components/LoadingSpinner.jsx:6]: Same here, I don't think there's a need to add the `uswds` prop here.

File: src/applications/combined-debt-portal/debt-letters/components/HistoryTable.jsx [hunk 3]
```diff
@@ -6,13 +6,13 @@ import { renderLetterHistory } from '../const/diary-codes';
 const HistoryTable = ({ history }) => {
   return (
     <div className=" vads-u-margin-y--4">
-      <va-table>
-        <va-table-row slot="headers">
+      <va-table uswds>
```
Comment (josephrlee) [inline src/applications/combined-debt-portal/debt-letters/components/HistoryTable.jsx:9]: Same here, `va-table` doesn't have a `uswds` prop available.

**Review by josephrlee — COMMENTED** [review]
I noticed a few components that don't need the `uswds` prop, namely the `va-telephone` and `va-loading` components.

**Comment by OptionSelect** [comment]
Breadcrumbs are back with new logic! 
image

**Comment by OptionSelect** [comment]
Corresponding HowDoIPay screenshot 
image

**Review by josephrlee — DISMISSED** [review]
Thanks for making those updates, looks good to me!

**Review by Dr-Pongo — DISMISSED** [review]
LGTM! 🚀 

Approving now with a note that I don't think `va-alert-expandable` has a `uswds` version, and we may want to have design take a peek at that natural disasters alert to see if we should leverage another component that's getting bumped to v3. 

CC: @josephrlee @mgleo

**Comment by josephrlee** [comment]
> LGTM! 🚀
> 
> Approving now with a note that I don't think `va-alert-expandable` has a `uswds` version, and we may want to have design take a peek at that natural disasters alert to see if we should leverage another component that's getting bumped to v3.
> 
> CC: @josephrlee @mgleo

Yup, there's no current `uswds` version. I think it's fine to keep it as is for now. I think the larger question is if we need it there at all. 

@denisecoveyduc, can you chime in here? Do we still need the Disaster Alert on the Debt Portal Overview page? If we don't, can we just comment it out/delete it? Or hide it under a feature flipper if we need to save it?

**Comment by denisecoveyduc** [comment]
@josephrlee Let's leave it in place at the moment. That was put in place by the Debt Management Center and I am not sure if it was a policy requirement or something more official.  I will reach out to DMC and make a new ticket should we want to remove or adjust. I created a ticket ([#75615](https://app.zenhub.com/workspaces/vsa---debt-607736a6c8b7e2001084e3ab/issues/gh/department-of-veterans-affairs/va.gov-team/75615)) and will reach out to DMC today. 
FYSA: @HeatherRienks @Tom-Davis

**Comment by OptionSelect** [comment]
image
All unit tests passing locally.

**Review by Dr-Pongo — APPROVED** [review]
LGTM! 🚀

**Pull request merged**